frontend/components/list-filter/list-filter.js

```diff
@@ -4,17 +4,17 @@ ActivitiesFrontend.ListFilter = function (container) {
   new MOJFrontend.FilterToggleButton({
     startHidden: true,
     toggleButton: {
-      container: $('.moj-action-bar__filter'),
+      container: document.getElementsByClassName('.moj-action-bar__filter'),
```

Comment:
Need to remove the dot at the start of the class name, or you could use query selector,

```document.querySelectorAll('.moj-action-bar__filter')```